test/integration/targets/vyos_interfaces/tests/cli/overridden.yaml

```diff
@@ -0,0 +1,50 @@
+---
+- debug:
+      msg: "START vyos_interfaces overridden integration tests on connection={{ ansible_connection }}"
+
+- include_tasks: _remove_config.yaml
+
+- include_tasks: _populate.yaml
+
+- name: Overrides all device configuration with provided configuration
+  vyos_interfaces: &overridden
+      config:
+          - name: eth0
```

Comment:
we shouldn't test against eth0, this is our mgmt interface

Reply:
Even though eth0 is mentioned, these tests actually do not modify any attributes of eth0. Given the way overridden is designed to work, if we don't specify eth0 in the task, it'll actually end up modifying its attributes. If you look at the commands generated by this task, you will not find any command that concerns eth0.